docs/content/preview/develop/change-data-capture/using-logical-replication/monitor.md

```diff
@@ -71,6 +71,33 @@ Provides a list of all replication slots that currently exist on the database cl
 | yb_stream_id | text | UUID of the CDC stream |
 | yb_restart_commit_ht | int8 | A uint64 representation of the commit Hybrid Time corresponding to the `restart_lsn`. This can be used by the client (like YugabyteDB connector) to perform a consistent snapshot (as of the `consistent_point`) in the case when a replication slot already exists. |
 
+### pg_stat_replication
```

Comment:
@aashir24-yb @vaibhav-yb This is only in preview (2.25)?

Reply:
Yes, this is only in preview so far.